src/Spec2-Core/SpTreeTablePresenter.class.st

```diff
@@ -322,6 +322,12 @@ SpTreeTablePresenter >> selection [
 	^ selectionMode value
 ]
 
+{ #category : #accessing }
```

Comment:
this is now category "api"
... and it needs a method comment (you can use the same as in `SpAbstractListPresenter>>#selecitonMode)` :)